lib/rouge/formatters/html_exercism.rb

```diff
@@ -128,12 +128,21 @@ def formatted_code(tokens, line_counter)
             formatted << end_line << start_line(line_counter)
           end
         end
-        formatted.gsub(start_line(line_counter), '')
+        adjust_final_lines formatted, line_counter
+      end
+
+      def adjust_final_lines(formatted, line_counter)
```

Comment:
(This is one of the reasons I'm in favour of revert and re-apply rather than patch-forwards.)